spec/classes/config_spec.rb

```diff
@@ -77,6 +77,19 @@
     }
   end
 
+  context 'log4j file' do
+    let(:params) { {
+      :log4j_file    => "/nail/etc/zookeeper/log4j.properties",
+    } }
+
+    it do
+      should contain_file('/etc/zookeeper/conf/zoo.cfg').with(
```

Comment:
Sorry for that. Fixed it.

Regards,
-- 
Raghavendra "Raghu" Prabhu
GPG Id : 0xD72BE977 | Fingerprint: B93F EBCB 8E05 7039 CD3C A4B8 A616 DCA1
D72B E977
wnohang.net   | about.me/raghavendra.prabhu


On Fri, Mar 4, 2016 at 5:24 PM, Kyle Anderson <notifications@github.com>
wrote:

> In spec/classes/config_spec.rb
> <https://github.com/Yelp/puppet-zookeeper/pull/2#discussion_r55059602>:
>
> > @@ -77,6 +77,19 @@
> >      }
> >    end
> >
> > +  context 'log4j file' do
> > +    let(:params) { {
> > +      :log4j_file    => "/nail/etc/zookeeper/log4j.properties",
> > +    } }
> > +
> > +    it do
> > +      should contain_file('/etc/zookeeper/conf/zoo.cfg').with(
>
> I'm surprised this passes, it is not correct
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Yelp/puppet-zookeeper/pull/2/files#r55059602>.
>